929/step1_2_for.py

```diff
@@ -0,0 +1,27 @@
+class Solution:
+    def numUniqueEmails(self, emails: List[str]) -> int:
+        def get_base_email(email):
+            local_name = []
+            domain_name = []
+            for char_ in email:
+
+                if local_name and local_name[-1] == "@":
```

Comment:
```suggestion
            for char_ in email:
                if local_name and local_name[-1] == "@":
```
スタイルガイドなどでfor文の直後に改行を入れないといったルールは見つけられませんでしたが、個人的にはfor文の直後に改行を入れるのではなく、変数を定義している箇所と分けるようにfor文の直前で改行するのが好みだなと感じました。

Reply:
ご指摘ありがとうございます、おっしゃるように、この部分の改行は不適切だったように思います。